desktop/Cargo.toml

```diff
@@ -52,7 +52,9 @@ rand = "0.8.5"
 thiserror.workspace = true
 
 [target.'cfg(target_os = "linux")'.dependencies]
-ashpd = "0.9.2"
+# The `tokio` and `async-std` features are mutually exclusive, and `ashpd` itself enables `tokio`
+# by default, while `rfd` enables `async-std` for it by default. So we need to disable `tokio` here.
+ashpd = { version = "0.10.2", default-features = false }
```

Comment:
Or should it be the other way around: Disable `async-std` and use `tokio`? Does it matter? I dunno... @kjarosh ?

See:
https://github.com/PolyMeilex/rfd/blob/5cb7002452da5d583f580c50f9753bc97117b1c7/Cargo.toml#L13-L21
https://github.com/bilelmoussaoui/ashpd/blob/62954f769c1ee7731fb59ed3a3d15c0410b82d2c/Cargo.toml#L14-L26
https://github.com/bilelmoussaoui/ashpd/blob/62954f769c1ee7731fb59ed3a3d15c0410b82d2c/src/lib.rs#L9-L10

Reply:
That's a good question... I see it's mainly used for IO stuff, i.e. detecting Flatpak. So maybe running this inside+outside of Flatpak and checking whether it works is enough

Reply:
To be sure, which one have we been using by default until now?

Reply:
For future reference: https://github.com/ruffle-rs/ruffle/pull/18645#issuecomment-2488319102